setup.cfg

```
@@ -50,6 +50,8 @@ calc =
     xrft
     xhistogram
 docs = sphinx >= 1.4
+3d_plot_k3d = k3d >= 2.8.0
+3d_plot_mayavi = mayavi >= 4.7.2
```

I'd prefer to not have too many options. User can still skip it, and just install what they need if the get the error. Providing a more or less trivial meta package for a single additional feels like over kill to me, but I am happy either way ...
```suggestion
3d_plot =
    k3d >= 2.8.0
    mayavi >= 4.7.2
    wand
```